### PR TITLE
fixed logging roles

### PR DESCRIPTION
**Pull request by paynejacob**
Removed edit from cluster members and allowed all users to view cluster level crds.

### CODE REVIEW

**Review by brendarearden — APPROVED** [review]
LGTM

**Review by cbron — COMMENTED** [review]
Can you make a ticket in 2.5.0 that explains the new goals to check for, and assign it to you and @izaac and then move to test after this merges.

File: packages/rancher-logging/overlay/templates/userroles.yaml [hunk 1]
```diff
@@ -15,31 +15,20 @@ rules:
...
     rbac.authorization.k8s.io/aggregate-to-view: "true"
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
 rules:
   - apiGroups:
       - "logging.banzaicloud.io"
     resources:
       - flows
       - outputs
+      - clusterflows
+      - clusteroutputs
```
Comment (cbron) [inline packages/rancher-logging/overlay/templates/userroles.yaml:31]: If we are giving view to all users, is there any sensitive info they could see on any 4 of these CRD's ? Or are they all in secrets.

Comment (izaac) [reply packages/rancher-logging/overlay/templates/userroles.yaml:31]: It is possible to store credentials/certificates as string values in the (Cluster)Outputs YAMLs. Although we're mounting from secrets using the UI.

**Review by cbron — APPROVED** [review]
We should document this caveat, and if we confirm any secrets are there we should add those to doc. Otherwise lgtm.

**Pull request merged**